packages/office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Example.scss

```diff
@@ -0,0 +1,28 @@
+:global {
+  .ms-ComboBoxExample {
+    > * {
+      margin-bottom: 20px; // add space between examples for readability
+    }
+
+    .ms-ComboBox {
+      max-width: 300px; // sets max width on field itself (not label)
+    }
+
+    .openButton {
+      display: block;
+      margin-top: 10px;
+    }
+
+    // This is for an example in ComboBox.Basic.Example.tsx which allows toggling some props
+    .ms-ComboBoxExample-toggles {
```

Comment:
Since you're moving scss around and it's not a lot of scss, could you inline all of these so it unblocks the Export to Codepen feature?

Reply:
What do you mean by inlining it? (an example in another file works)

Reply:
I find @Vitalius1's PRs well done, especially those involving mergeStyles conversion such as https://github.com/OfficeDev/office-ui-fabric-react/pull/7376/files. Personally I might try to do that in a separate PR from this. Up to you!

Reply:
Not sure if the 3 of us here are on the same page as to what style conversion work is required here but it is a very trivial ask here, and nowhere on the scale that Vitalie has been executing on his PRs....